src/network/server_system.cpp

```diff
@@ -63,48 +67,31 @@ void ServerSystem::Init()
     {
         L1 ( "An error occurred while trying to create an ENet server host.\n" );
     }
+    mThreaded = Settings::Get().GetBool( "network.threaded", true );
+    if (mThreaded)
+    {
+        mRunning = true;
```

Comment:
I'd update the mRunning regardless of threading, otherwise the name and run-time value is a bit misleading in non-mt network mode.